tests/test_tacacs_server_host.rb

```diff
@@ -165,6 +187,8 @@ def test_tacacsserverhost_get_default_port
   end
 
   def test_tacacsserverhost_set_port
+    return if platform == :ios_xr
```

Comment:
As discussed, aren't these the ones you were going to change to use `validate_property_excluded?` and assert for unsupported?